### ISSUE TITLE
automatically run scripts before making a commit

### ISSUE THREAD

**Issue by hassnian** [kodadot/nft-gallery#11015]
after #11009

> would be nice if the script automatically ran before making a commit

maybe in new pr?

```ts
  "lint-staged": {
    "*": [
      "pnpm lint:fix",
      "pnpm locales:order"
    ]
  },
```

_Originally posted by @roiLeo in https://github.com/kodadot/nft-gallery/issues/11009#issuecomment-2360818645_

**Comment by roiLeo** [comment]
feel free to assign me
